airbyte-integrations/connectors/source-mongodb-v2/metadata.yaml

```diff
@@ -8,7 +8,7 @@ data:
   connectorSubtype: database
   connectorType: source
   definitionId: b2e713cd-cc36-4c0a-b5bd-b47cb8a0561e
-  dockerImageTag: 1.4.3
+  dockerImageTag: 1.4.4
```

Comment:
We should do a minor version increase here

Reply:
yes, fixed.